MANIFEST.in

```diff
@@ -20,6 +20,9 @@ include *.toml
 
 include .bumpversion.cfg
 
+include papermill/tests/fixtures/foo-0.0.1.dist-info/METADATA
```

Comment:
This looks strange to be included, is it testing artifact?

Reply:
Yes, it's a testing file -- we provide just enough wheel metadata in that path to give us one entry point.

Reply:
@Borda any opinion on this?

Reply:
I think that test shall not be distributed, so for testing you can install it with `-e` so if shall see all local files too

Reply:
@Borda It literally caused a GitHub Actions failure when I didn't include it, I'll note.